resources/public/js/cookies.js

```diff
@@ -0,0 +1,27 @@
+function Pop(){var cssRuleFile="/css/cookiestyles.css";let lnk=document.createElement("link");lnk.setAttribute("rel","stylesheet");lnk.setAttribute("type","text/css");lnk.setAttribute("href",cssRuleFile);document.getElementsByTagName("head")[0].appendChild(lnk);let styl="undefined";var conDivObj;var fadeInTime=10;var fadeOutTime=10;let cookie={name:"cookieconsent_status",path:"/",expiryDays:365*24*60*60*5000};let content={message:"This website uses cookies to ensure you get the best experience on our website.",btnText:"Got it!",mode:"  banner bottom",theme:" theme-classic",palette:" palette1",link:"Learn more",href:"https://www.cookiesandyou.com",target:"_blank"};let createPopUp=function(){if(typeof conDivObj==="undefined"){conDivObj=document.createElement("DIV");conDivObj.style.opacity=0;conDivObj.setAttribute("id","spopupCont");}
```

Comment:
Renaming the div's id `spopupCont` to `cookie-policy-popup` fixes the issues with adblocker